Project.toml

```diff
@@ -0,0 +1,6 @@
+name = "Suppressor"
+uuid = "fd094767-a336-5f1f-9728-57cf17d0bbfb"
+version = "0.2.0"
```

Comment:
May be worth just jumping to `1.0.0` as this package hasn't seen many changes lately and seems rather stable.

Reply:
The registered version is `0.1.1`. 

If I'm reading correctly, this PR doesn't change much except dropping Julia 0.6. In which case, please tag it as `0.1.2`, even if you also tag it as `1.0` the day after. 

The reason I suggest this is that it currently bounds Compat.jl to <=2, here:
https://github.com/JuliaRegistries/General/blob/master/S/Suppressor/Compat.toml
If the end of any chain of packages depends on Suppressor@0.1, and I need Compat@3, then these can't be installed together.

Reply:
> please tag it as 0.1.2

Due to changing the minimum required version of Julia this has to be at least a minor release. This is a requirement when registering packages.

Reply:
Are you sure? I could be misinformed, but I thought e.g. this should be read as dropping Julia 0.6 on v0.3.1:
https://github.com/JuliaRegistries/General/blob/master/A/AxisArrays/Compat.toml

Reply:
I'm very confident: https://github.com/JuliaRegistries/RegistryCI.jl/blob/6b4ba393274dc8b3d5472751dc644d792697a956/src/AutoMerge/guidelines.jl#L79

It's actually hard to see this in General PRs as the comments are removed once the block has been resolved.